### PR TITLE
Reader bugfixes & show display name of content creator

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

**Comment by nwingt** [comment]
Don't merge on GitHub

File: app/models/content/content.ts [hunk 1]
```diff
@@ -67,6 +68,7 @@ export const ContentModel = types
       } finally {
         self.isFetchingDetails = false
         self.hasFetchedDetails = true
+        self.hasCached = true
```
Comment (williamchong007) [inline app/models/content/content.ts]: Not sure if want to set `hasCached` on error cases

Comment (williamchong007) [reply app/models/content/content.ts]: then the expect case for error is to always have `isLoading === true`?

**Comment by williamchong007** [comment]
@nwingt is it expected for error case to forever stuck in loading? Will it recover in any cases except app restart?

**Comment by nwingt** [comment]
> @nwingt is it expected for error case to forever stuck in loading? Will it recover in any cases except app restart?

@williamchong007 Resolve by adding `hasFetchedDetails`

**Pull request merged**